### PR TITLE
Archiving

### PR DESCRIPTION
**Pull request by ppouyaa**
If there is anything wrong with the comments I will fix it later. this is just to see if the archiving is the way you wanted.

### CODE REVIEW

**Comment by light-weaver** [comment]
There are duplicate files in the request.

**Comment by ppouyaa** [comment]
Removed the duplicate files

File: desdeo_problem/problem/Problem.py [hunk 1]
```diff
@@ -1284,6 +1309,18 @@ def train_one_objective(
             msg = "Support for VectorDataObjective not supported yet"
             raise ProblemError(msg)
 
+    def evaluate(self, decision_vectors: np.ndarray, use_surrogate: bool) -> EvaluationResults:
+
+        names = np.hstack((self.variable_names,self.objective_names))
```
Comment (light-weaver) [inline desdeo_problem/problem/Problem.py:1314]: this could be self.archive.columns

File: desdeo_problem/problem/Problem.py [hunk 2]
```diff
@@ -1186,9 +1207,12 @@ def __init__(
             raise ProblemError(msg)
         # TODO: Implement the rest
         objectives = []
-        for obj in objective_names:
+        self.archive = data # this is for model management to archive the solutions and decision variables
+        #check if evaluator is NOne in that case make a list of nones and the lenght of the list is the same as obj_names
```
Comment (light-weaver) [inline desdeo_problem/problem/Problem.py:1211]: This should _reeeealy_ be done

File: desdeo_problem/problem/Problem.py [hunk 3]
```diff
@@ -1186,9 +1207,12 @@ def __init__(
...
+        #check if evaluator is the same lenght as obj_names if not rais a problem error
```
Comment (light-weaver) [inline desdeo_problem/problem/Problem.py:1212]: This should _reeeeealy_ be done

**Review by light-weaver — APPROVED** [review]

**Pull request merged**